README.md

```diff
@@ -27,6 +27,38 @@ create forms by leveraging the capabilities of JSON and JSON schema.
     <jsonforms schema="vm.schema" uischema="vm.uiSchema" data="vm.data"></jsonforms>
   </div>
   ```
+6. Use Bootstrap version:
+
+  In order to use the bootstrap version you need to load the `bootstrap.css`, furthermore you need the `jsonforms-bootstrap.css` instead of the `jsonforms.css`:
```

Comment:
Update to: In order to use the Bootstrap version of JSONForms, you'll need to first load the CSS file of Bootstrap, e.g. by [installing it via npm](https://www.npmjs.com/package/bootstrap) and referencing it in your `index.html`. Furthermore, you'll also need to load JSONForms specific CSS stylings bits which are contained in the `jsonforms-bootstrap.css` file.